modules/imaging_browser/php/viewsession.class.inc

```diff
@@ -774,5 +774,19 @@ class ViewSession extends \NDB_Form
         );
     }
 
+    /**
+     * Generate a breadcrumb trail for that page.
+     *
+     * @return \LORIS\BreadcrumbTrail The trail.
+     */
+    public function getBreadcrumbs(): \LORIS\BreadcrumbTrail
+    {
+        $label = ucwords(str_replace('_', ' ', $this->name));
+        return new \LORIS\BreadcrumbTrail(
+            new \LORIS\Breadcrumb($label, "/$this->name"),
+            new \LORIS\Breadcrumb('View Session', "/$this->name/$this->page")
```

Comment:
How come the format here is different than it was in the dicom archive where it used the $_SERVER superglobal? Isn't this going to lose the relevant get params (ie. sessionID)